packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -983,6 +960,28 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends React.Componen
       return '';
     }
   }
+
+  /**
+   * This should be called when the user does something other than use text entry to trigger suggestions.
+   *
+   */
+  private _userTriggeredSuggestions = () => {
```

Comment:
👍  for moving this. Also a valuable refactor target in the future as there are tons of paths through these branches.